### PR TITLE
added missing banner in readme

### PR DESCRIPTION
**Pull request by kunalrustagi08**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +20,7 @@ We love to hear from you so if you have questions, comments or find a bug in the
 
 - Open an issue on this repository
 - Tweet at us! We're [@NexmoDev on Twitter](https://twitter.com/NexmoDev)
-- Or [join the Nexmo Community Slack](https://developer.nexmo.com/community/slack)
+- Or join the [Nexmo Community on Slack](https://developer.nexmo.com/community/slack)
```
Comment (michaeljolley) [inline README.md]: The wording here is fine. We actually refer to it as the Community Slack.

However, the references to Nexmo do need to change to Vonage. (i.e. the twitter handle and Vonage Community Slack)

**Review by michaeljolley — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kunalrustagi08) [reply README.md]: Does Vonage Community Slack channel has any new invite link?

Comment (michaeljolley) [reply README.md]: The link is the same, but the Twitter account is VonageDev

**Review by michaeljolley — CHANGES_REQUESTED** [review]
Can we back the "us at " out?

**Comment by kunalrustagi08** [comment]
sorry for the mistake

**Review by michaeljolley — APPROVED** [review]
Thanks @kunalrustagi08! Looks good.

bitmoji

**Pull request merged**